README.md

```diff
@@ -1,5 +1,4 @@
-# PDF.js [![Build Status](https://travis-ci.org/mozilla/pdf.js.svg?branch=master)](https://travis-ci.org/mozilla/pdf.js)
-
```

Comment:
Please don't remove blank lines, since they aid with readability of the "raw"-file.